bluesky/bundlers.py

```diff
@@ -679,13 +742,7 @@ async def collect(self, msg):
             objs_read = frozenset(ev["data"])
             stream_name, descriptor_uid = local_descriptors[objs_read]
             seq_num = next(self._sequence_counters[stream_name])
-
             event_uid = new_uid()
-
-            reading = ev["data"]
-            for key in ev["data"]:
-                reading[key] = reading[key]
```

Comment:
This seems like a relic from when numpy arrays were turned into lists:
https://github.com/bluesky/bluesky/blame/9ed5d21acc7509bd61a663ea06c6a0ede58a8312/bluesky/run_engine.py#L1687

I deleted it